testing/features/support/drivers/browserstack.rb

```diff
@@ -14,7 +14,15 @@ def register
     def register_browser
       validate_driver_request
 
-      browser_type_class.new.register
+      # This is missing from the selenium repo proper. It has been added to all relevant spots
+      # HERE: https://github.com/SeleniumHQ/selenium/pull/9862
+      # Remove once we are only supporting Selenium 4
+      #
+      # LH - Sep '21
+
+      require "selenium/webdriver/common/platform"
+
+      CaTesting::Drivers::Browserstack.new(browser, configurable_capabilities).register
```

Comment:
This will need a new method here to push in the device specific caps but only when they are provided